chromium_src/ui/native_theme/native_theme.cc

```diff
@@ -14,10 +14,9 @@
 namespace ui {
 
 SkColor NativeTheme::GetSystemButtonPressedColor(SkColor base_color) const {
-  const SkColor kBraveColorBrand = SkColorSetRGB(0xfb, 0x54, 0x2b);
+  const SkColor kBraveColorBrand =SkColorSetRGB(0xff, 0x54, 0x2b);
```

Comment:
what is this colour change for?

Reply:
reverted to the default buttons color, and KbraveBrand color

Reply:
revert kara hai toh ye changed nahi ana chaiye na?

Reply:
mtlb bhaiya manually change kr diya hai, jo phele tha whi color mai

Reply:
manually bhi revert hua hai toh ye change nahi ana chaiye

Reply:
okay bhaiya looking into it

Reply:
fixed it bhaiya, it was the typo